frontend/src/components/DetailsTable.js

```diff
@@ -35,13 +44,33 @@ const DetailsTable = ({ data }) => {
     createData('Tax Year:', data.tax_year),
     createData(
       'Consent to Disclosure and Storage of, and Access to, Personal Information:',
-      createConsentValue(data.consent_personal, data.first_name, data.last_name, data.created)
+      createConsentValue(
+        data.consent_personal,
+        data.first_name,
+        data.last_name,
+        data.created
+      )
     ),
     createData(
       'Consent to Disclosure of Information from Income Tax Records:',
-      createConsentValue(data.consent_tax, data.first_name, data.last_name, data.created)
+      createConsentValue(
+        data.consent_tax,
+        data.first_name,
+        data.last_name,
+        data.created
+      )
     )
   ];
+  const dlIndex = rows.findIndex((object) => {
```

Comment:
I would create this data conditionally in the array, leaving it undefined otherwise and then filter out empty values to avoid any string changing issues.

Reply:
`data.drivers_licence && createData("B.C. Driver's Licence number:", data.drivers_licence)`